completions/ip

```diff
@@ -186,7 +186,7 @@ _comp_cmd_ip()
                     elif [[ $prev == scope ]]; then
                         _comp_cmd_ip__iproute2_etc rt_scopes
                     else
-                        : # TODO
+                        _comp_compgen -- -W 'dev scope home nodad mngtmpaddr noprefixroute autojoin'
```

Comment:
Should we include all the words in `IFADDR`  as well? Ref https://github.com/iproute2/iproute2/blob/89210b9ec1c445ae963d181b5816d12a0cdafbb6/ip/ipaddress.c#L63-L66

Reply:
Yeah we should, will do that soon

Reply:
Added them and pushed, should be ready